src/components/Log/LogMessageAccordion.tsx

```diff
@@ -0,0 +1,70 @@
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Accordion, AccordionDetails, AccordionSummary, Box, BoxProps, Typography } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { MessageItem } from '../../pages/messages/components/MessageList';
+import { LogEntryType } from '../../types/log.types';
+import { Message } from '../../types/publication_types/ticket.types';
+import { dataTestId } from '../../utils/dataTestIds';
+import { logBackgroundColor } from './LogEntry';
+
+interface LogMessageAccordionProps {
+  messages: Message[];
+  type: LogEntryType;
+}
+
+export const LogMessageAccordion = ({ messages, type }: LogMessageAccordionProps) => {
+  const { t } = useTranslation();
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <Accordion
```

Comment:
`Accordion` er litt ekkel ved at den legger til en del styling selv, og justerer en del innhold selv basert på om den er åpen eller ikke. I vårt tilfelle her lurer jeg på om vi kanskje ikke vil at det skal være en divider bare når den ikke åpne, og at den skal legge til mer padding for å vise at den er åpen? Det er primært gjort for at det skal se bedre ut når det er flere Accordions rett etter hverandre, som forsåvidt er normalen for accordion.

![bilde](https://github.com/user-attachments/assets/c9fa8cb8-8325-4065-a91b-687611866e93)
![bilde](https://github.com/user-attachments/assets/b5022a11-7415-475c-9351-69a0f8b715a6)

Lurer på om vi heller kanskje vil ha `<Collapse>` her, som fortsatt tillater åpning/lukking, men ikke legger til mye annen ekstra styling. Vi har eksempler som ligner veldig på det i `DoiRequestAccordion.tsx` og `MoreActionsCollapse.tsx`. I den siste har vi ikke brukt Collapse en gang, så ikke sikkert vi trenger det heller, men tror man får fade og litt sånn ekstra snacks med Collapse